src/auth0/handlers/connections.js

```diff
@@ -31,6 +31,25 @@ export default class ConnectionsHandler extends DefaultHandler {
     return super.objString({ name: connection.name, id: connection.id });
   }
 
+  getFormattedOptions(connection, clients) {
+    try {
+      return {
+        options: {
+          ...connection.options,
+          idpinitiated: {
+            ...connection.options.idpinitiated,
+            client_id: convertClientNameToId(
+              connection.options.idpinitiated.client_id,
+              clients
+            )
+          }
+        }
+      };
+    } catch (e) {
+      return {};
```

Comment:
maybe this catch is covering up some error that we don't expect, for example, if connection.options is not iterable. I just put this comment here to be sure you had this in mind.

Reply:
Thanks @luisbritos for the review. The `try` block is intended to catch all exceptions, including if `connection.options` is not iterable. The `catch` block will simple return empty object such that no field from the `connection` object will be overridden.